conda_smithy/templates/build_steps.sh.tmpl

```diff
@@ -25,15 +27,20 @@ conda-build:
  root-dir: ${FEEDSTOCK_ROOT}/build_artifacts
 
 CONDARC
-
-{%- if build_with_mambabuild %}
-GET_BOA=boa
-BUILD_CMD=mambabuild
+{% if build_with_mambabuild %}
+{%- set CONDA_INSTALL_CMD="mamba" %}
+{%- set GET_BOA="boa " %}
+{%- set BUILD_CMD="mambabuild" %}
 {%- else %}
-BUILD_CMD=build
+{%- set CONDA_INSTALL_CMD="mamba" %}
+{%- set GET_BOA="" %}
+{%- set BUILD_CMD="build" %}
 {%- endif %}
 
-conda install --yes --quiet "{{ remote_ci_setup }}" conda-build pip ${GET_BOA:-} -c conda-forge
+{{ CONDA_INSTALL_CMD }} install --yes --quiet "{{ remote_ci_setup }}" conda-build pip {{ GET_BOA }}-c conda-forge
+{%- if build_with_mambabuild %}
+{{ CONDA_INSTALL_CMD }} update --yes --quiet "{{ remote_ci_setup }}" conda-build pip {{ GET_BOA }}-c conda-forge
+{%- endif %}
```

Comment:
```suggestion
{{ CONDA_INSTALL_CMD }} install --update-specs --yes --quiet "{{ remote_ci_setup }}" conda-build pip {{ GET_BOA }}-c conda-forge
{%- if build_with_mambabuild %}
{{ CONDA_INSTALL_CMD }} update --update-specs --yes --quiet "{{ remote_ci_setup }}" conda-build pip {{ GET_BOA }}-c conda-forge
{%- endif %}
```

Reply:
Sorry to bother you @isuruf, but can you look at this? I think with this commit we should be future-proof and good to merge.

Reply:
Did you try this in a feedstock?

Reply:
Nope but I will go ahead with that.